scholarx/archive/index.html

```diff
@@ -77,15 +77,15 @@ <h6>COHORT</h6>
         <hr>
         <div class="mt-2 mb-5">
             <button class="btn btn-sm custom-checkbox mb-3 btn-checkbox">
-                <input class="custom-control-input" id="chk2019" type="checkbox">
+                <input class="custom-control-input" id="chk2019" type="checkbox" onclick="cohort()">
                 <label class="custom-control-label" for="chk2019">2019</label>
             </button>
             <button class="btn btn-sm custom-checkbox mb-3 btn-checkbox">
-                <input class="custom-control-input" id="chk2020" type="checkbox">
+                <input class="custom-control-input" id="chk2020" type="checkbox" onclick="cohort()">
```

Comment:
Let's change the labels.
```suggestion
                <input class="custom-control-input" id="2020" type="checkbox" onclick="cohort()">
```